{{cookiecutter.role_name}}/molecule/resources/provisioning/WINRM/destroy.yml

```diff
@@ -0,0 +1,33 @@
+---
```

Comment:
WINRM/destroy.yml will be same as EPC/destroy.yml.
Will weagree with duplication?

Reply:
Agree as discussed